bulk-model-sync-lib/src/commonMain/kotlin/org/modelix/model/sync/bulk/Util.kt

```diff
@@ -21,6 +21,21 @@ import org.modelix.model.api.BuiltinLanguages
 import org.modelix.model.data.ModelData
 import org.modelix.model.data.NodeData
 
+/**
+ * Helper-method used by the gradle plugin and the MPS solution.
+ * Checks if a module is included in the sync.
```

Comment:
```suggestion
 * Checks if a module is included in the sync.
```

The first line of a docstring should usually state what the method does. The indication, where it is used, will easily become outdated when new usages appear or disappear. And the IDE probably does a better job finding them.